components/mod-event/EventItem.tsx

```diff
@@ -152,20 +154,27 @@ const TakedownOrMute = ({
 const EventLabels = ({
   header,
   labels,
+  isTag = false,
 }: {
   header: string
   labels?: string[]
+  isTag?: boolean
 }) => {
   if (!labels?.length) return null
   return (
     <LabelList>
       <span className="text-gray-500 dark:text-gray-50">{header}</span>
       {labels.map((label) => {
-        const labelGroup = getLabelGroupInfo(unFlagSelfLabel(label))
+        if (isTag) {
+          return <LabelChip key={label}>{label}</LabelChip>
+        }
+        // Moderation events being displayed means that these events were added by the current service
+        // so we can assume that the src is the same as the configured ozone service DID
         return (
-          <LabelChip key={label} style={{ color: labelGroup.color }}>
-            {displayLabel(label)}
-          </LabelChip>
+          <ModerationLabel
+            key={label}
+            label={{ val: label, src: OZONE_SERVICE_DID, uri: '', cts: '' }}
+          />
```

Comment:
I don't think we'll be able to capture the "current service" using a constant, since it's determined at runtime.  I think there's a `useSession()` hook kicking around that we could use to grab it, though.

Reply:
we can also just get it from `client` directly, right?